docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -86,10 +86,22 @@
     "The <sup>3</sup>He opacity is a cell-specific parameter and can vary after each filling process. Hence, for precise data correction, it must be measured for each newly filled cell. It linearly depends on the neutron wavelength as follows:\n",
     "\n",
     "$$\n",
+    "O(\\lambda)=n\\cdot \\sigma_0\\cdot l \\cdot\\lambda\n",
+    "$$\n",
+    "\n",
+    "with n being the number density of <sup>3</sup>He gas in the cell, $\\sigma_0$ the absorption cross section of 1 $\\mathring A$ neutrons by <sup>3</sup>He ($\\sigma_0 = 2966 \\cdot 10^{−24} cm^2$), and the path length through the cell $l$ [Ref. 1]. As <sup>3</sup>He can be considered to follow the ideal gas law, $n$ can be replaced via $p=n\\cdot k_B\\cdot T$:\n",
```

Comment:
Do we have an error bar for the `2966`? Then we could include it in the propagation of uncertainties.